pkg/controller/migcluster/validation.go

```diff
@@ -205,3 +226,37 @@ func (r ReconcileMigCluster) testConnection(cluster *migapi.MigCluster) error {
 
 	return nil
 }
+
+func (r *ReconcileMigCluster) saIsClusterAdmin(cluster *migapi.MigCluster) (bool, error) {
+	// check for access to all verbs on all resources in all namespaces
+	// in order to determine if the service account has cluster-admin
+	attributes := auth.ResourceAttributes{
+		Group:    "*",
+		Resource: "*",
+		Verb:     "*",
+		Version:  "*",
+	}
+
+	sar := auth.SelfSubjectAccessReview{
+		Spec: auth.SelfSubjectAccessReviewSpec{
+			ResourceAttributes: &attributes,
```

Comment:
Separate variable is fine but could be defined here _inline_ since it's not used anywhere else.